ci/ray_ci/builder_container.py

```diff
@@ -18,22 +18,24 @@ class PythonVersionInfo(TypedDict):
 
 
 class BuilderContainer(Container):
-    def __init__(self, python_version: str) -> None:
+    def __init__(self, python_version: str, build_type: str = "optimized") -> None:
```

Comment:
is there still any place calling this without `build_type`? maybe remove the default value?